### PR TITLE
Hcl beta tag

### PR DESCRIPTION
**Pull request by SwampDragons**
This updates the website to make sure it no longer refers to HCL2 templates as in beta

### CODE REVIEW

**Review by nywilken — APPROVED** [review]
Looks great!

File: website/content/partials/from-1.5/beta-hcl2-note.mdx [hunk 1]
```diff
@@ -1,6 +1,7 @@
--> **Note:** This page is about HCL2 in Packer 1.5 and later. HCL2 support for
-Packer is still in Beta. Please see the [Packer Issue
-Tracker](https://github.com/hashicorp/packer/issues/9176) for a list of
-supported features. For the old-style stable configuration language see
-[template docs](/docs/templates). You can now transform your JSON file into an
-HCL2 config file using the [hcl2_upgrade command](/docs/commands/hcl2_upgrade).
+-> **Note:** This page is about HCL2 Packer templates. HCL2 templates
```
Comment (sylviamoss) [inline website/content/partials/from-1.5/beta-hcl2-note.mdx:1]: Nice!! I like better this note 😏

**Review by sylviamoss — APPROVED** [review]
Very happy to remove the BETA flag 👏🏼 
LGTM! 👍🏼

File: website/content/partials/from-1.5/legacy-json-warning.mdx [hunk 2]
```diff
@@ -0,0 +1,6 @@
+-> **Note:** This page is about older-style JSON Packer templates. JSON
```
Comment (sylviamoss) [inline website/content/partials/from-1.5/legacy-json-warning.mdx:1]: I like how you called `older-style` 👍🏼

**Review by azr — APPROVED** [review]
Nice one !

**Pull request merged**